site/content/features/records.md

```diff
@@ -162,12 +232,12 @@ public class RecordDemo {
 {{< /sandboxsource >}}
 {{< /sandbox >}}
 
-You can also add code to the body of the canonical constructor. When you do so, you don't repeat the parameter names and types:
+You can also add code to the body of the canonical constructor. When you do so, you can declare the constructor in the usual way:
 
 {{< sandbox version="java16" mainclass="RecordDemo" >}}
 {{< sandboxsource "RecordDemo.java" >}}
 record Point(double x, double y) {
-   public Point {
+   public Point(double x, double y) {
```

Comment:
When I run this example I get the following error (on 16-ea+29-2091):

```
RecordDemo.java:4: error: variable x might not have been initialized
   }
   ^
```

Reply:
Thanks, I misunderstood how the explicit form works. I pushed the change to this branch. Do I need another pull request?

Reply:
Great! No need for a new pull request. You can keep pushing commits until you're happy with the result ;)

I'll squash it into a singe commit when I merge it to master.